packages/addons/addon-depends/docker/moby/patches/moby-001-user-addon-storage-location.patch

```diff
@@ -0,0 +1,279 @@
+diff -Naur a/cmd/dockerd/daemon_unix.go b/cmd/dockerd/daemon_unix.go
```

Comment:
could it be done with some sed ? avoids patching and is likely more future proof

Reply:
The patch is created using a sed command. I'd rather use a patch so we know what changes between versions.

Reply:
I've added the sed command as a comment in the patch (like I had when I originally created the patch when I originally added docker).